sys/auto_init/saul/auto_init_tsl2561.c

```diff
@@ -28,42 +28,44 @@
 /**
  * @brief   Define the number of configured sensors
  */
-#define TSL2561_NUMOF    (sizeof(tsl2561_params) / sizeof(tsl2561_params[0]))
+#define TSL2561_NUM      (sizeof(tsl2561_params) / sizeof(tsl2561_params[0]))
```

Comment:
why this renaming? Its `I2C_NUMOF`, too?

Reply:
Fixed